### PR TITLE
Add dashboard documentation

### PR DESCRIPTION
**Pull request by jfaz1**
Feel free to make any requests/changes

### CODE REVIEW

**Comment by cxxxr** [comment]
@jfaz1  Set up to be able to write from the GitHub’s manage access screen.

**Comment by jfaz1** [comment]
Thanks :smile:

File: content/en/modes/dashboard.md [hunk 1]
```diff
@@ -0,0 +1,134 @@
...
+### Creating your own `dashboard-item`
+
+Creating a custom `dashboard-item` is straightforward; just derive the class and implement the `draw-dashboard-item` method.
+
+You are responsible for the actual insertion of text inside of `draw-dashboard-item`. There is a convenience function, `create-centered-string`, which you can use when inserting text.
+
+As an example, let's make a `dashboard-item` that fetches a random fact from an HTTP API and prints it:
+
+```lisp
+(defclass dashboard-random-fact (dashboard-item)
```
Comment (vindarel) [inline content/en/modes/dashboard.md:97]: you'll need to show which package we are in.

Comment (jfaz1) [reply content/en/modes/dashboard.md:97]: Done, thanks!

**Pull request merged**